nuxeo-distribution/nuxeo-functional-tests/src/main/java/org/nuxeo/functionaltests/RestHelper.java

```diff
@@ -401,14 +410,12 @@ public static int countQueryPageProvider(String providerName) {
     /**
      * @since 9.2
      */
-    public static String createDirectoryEntry(String directoryName, Map<String, String> properties) {
+    public static DirectoryEntry createDirectoryEntry(String directoryName, Map<String, String> properties) {
```

Comment:
I would prefer to not propagate java client objects outside of this class. Could you set back `String` and update `updateDirectoryEntry(String, DirectoryEntry)` to `updateDirectoryEntry(String, String, Map<String, String>)` ? Here the code:
```
    public static void updateDirectoryEntry(String directoryName, String entryId, Map<String, String> properties) {
        DirectoryEntry entry = new DirectoryEntry();
        entry.setProperties(properties);
        entry.putIdProperty(entryId);
        CLIENT.directoryManager().directory(directoryName).updateEntry(entry);
    }
```